base/shell/explorer/precomp.h

```diff
@@ -149,6 +150,7 @@ DECLARE_INTERFACE_(ITrayWindow, IUnknown)
     STDMETHOD_(HWND, DisplayProperties) (THIS) PURE;
     STDMETHOD_(BOOL, ExecContextMenuCmd) (THIS_ UINT uiCmd) PURE;
     STDMETHOD_(BOOL, Lock) (THIS_ BOOL bLock) PURE;
+    STDMETHOD_(BOOL, IsTaskWnd) (THIS_ HWND hWnd) PURE;
```

Comment:
I am not sure if this is an existing interface we are mimicking here, or that it is some ReactOS specific thing?
If it's an existing interface, then you can't just add a function to it.

Reply:
`ITrayWindow` is ReactOS-specific and non-public interface.